cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/efs/AwsEfsFileSystem.java

```diff
@@ -53,9 +55,17 @@
 
     private String throughputMode;
 
+    private List<String> associatedInstanceGroupNames;
+
     public AwsEfsFileSystem(String backupPolicyStatus, boolean encrypted, String fileSystemPolicy, Map<String, String> fileSystemTags, String kmsKeyId,
```

Comment:
Not a big deal but this constructor is taking 9 arguments. Perhaps it would make sense to use the Builder pattern instead to have a cleaner code.

Reply:
+1

Reply:
done